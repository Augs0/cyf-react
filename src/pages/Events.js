import React from 'react';
import { TopSection } from '../components/TopSection'
import { Event } from '../components/Event'

const SomeEvents = [
  {
    id: '1',
    date: 'Monday, Jun 19',
    time: '12:00 PM',
    location: 'Ticketmaster (Angel, London)',
    title: 'Javascript Module 1',
    mentors: ['Kash'],
    sponsors: ['Ticketmaster']
  },
  {
    id: '2',
    date: 'Tuesday, Jun 20',
    time: '12:00 PM',
    location: 'Ticketmaster (Angel, London)',
    title: 'Javascript Module 1',
    mentors: [{name: 'Kash', url: ''}],
    sponsors: [{name: 'Ticketmaster', url: ''}]
  }
]

const renderEvents = () => {
    return (
      SomeEvents.map((event) => {
        return (
          <Event key={event.id} {...event} />
        )
      })
    )
  }

export const Events = () => {
  return (
    <div>
      <TopSection
        title="Events"
        content="Our current classes are done every Sunday in London and Edinburgh"/>
      <div className="col-sm-8 col-sm-offset-2 section-description">
        {renderEvents()}
      </div>
    </div>
  )
}