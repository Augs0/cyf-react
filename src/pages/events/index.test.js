import React from 'react';
import { shallow } from 'enzyme';

import Events from './';

describe('Events Component', () => {
  it('renders', () => {
    expect(shallow(<Events />)).toMatchSnapshot();
  });
});
