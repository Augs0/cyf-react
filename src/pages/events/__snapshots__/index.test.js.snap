// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Events Component renders 1`] = `
<Events
  events={Array []}
/>
`;
